x-pack/plugins/rollup/public/crud_app/sections/job_create/step_logistics/step_logistics.js

```diff
@@ -0,0 +1,285 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { Component, Fragment } from 'react';
+import { injectI18n, FormattedMessage } from '@kbn/i18n/react';
+
+import {
+  EuiButtonEmpty,
+  EuiCallOut,
+  EuiFieldNumber,
+  EuiFieldText,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiForm,
+  EuiFormHelpText,
+  EuiFormRow,
+  EuiIcon,
+  EuiSpacer,
+  EuiText,
+  EuiTitle,
+} from '@elastic/eui';
+
+import { logisticalDetailsUrl } from '../../../services';
+
+export class StepLogisticsUi extends Component {
+  render() {
+    const {
+      fields,
+      onFieldsChange,
+      showStepErrors,
+      fieldErrors,
+    } = this.props;
+
+    const {
+      id,
+      indexPattern,
+      rollupIndex,
+      rollupCron,
+      rollupPageSize,
+    } = fields;
+
+    const {
+      id: errorId,
+      indexPattern: errorIndexPattern,
+      rollupIndex: errorRollupIndex,
+      rollupCron: errorRollupCron,
+      rollupPageSize: errorRollupPageSize,
+    } = fieldErrors;
+
+    const illegalCharacters = ['\\', '/', '?', '"', '<', '>', '|'].join(', ');
+
+    return (
+      <Fragment>
+        <EuiFlexGroup justifyContent="spaceBetween">
+          <EuiFlexItem grow={false}>
+            <EuiTitle>
+              <h3>
+                <FormattedMessage
+                  id="xpack.rollupJobs.create.stepLogistical.title"
+                  defaultMessage="Logistics"
+                />
+              </h3>
+            </EuiTitle>
+
+            <EuiText>
+              <p>
+                <FormattedMessage
+                  id="xpack.rollupJobs.create.stepLogistical.description"
+                  defaultMessage="Define the manner in which data will be rolled up."
+                />
+              </p>
+            </EuiText>
+          </EuiFlexItem>
+
+          <EuiFlexItem grow={false}>
+            <EuiButtonEmpty
+              size="s"
+              flush="right"
+              href={logisticalDetailsUrl}
+              target="_blank"
+              iconType="help"
+            >
+              <FormattedMessage
+                id="xpack.rollupJobs.create.stepLogistical.readDocsButton.label"
+                defaultMessage="Read the docs"
+              />
+            </EuiButtonEmpty>
+          </EuiFlexItem>
+        </EuiFlexGroup>
+
+        <EuiSpacer size="l" />
+
+        <EuiForm>
+          <EuiFormRow
+            label={(
+              <FormattedMessage
+                id="xpack.rollupJobs.create.stepLogistical.fieldName.label"
+                defaultMessage="Rollup job name"
```

Comment:
Good catch. I'll update the text to refer to it as an "ID" since this is how the docs and API refer to it, check the acceptable values with Zach, and implement client-side validation.